src/anomalib/data/depth/folder_3d.py

```diff
@@ -129,17 +132,23 @@ def make_folder3d_dataset(
             ].image_path.to_numpy()
 
         # make sure every rgb image has a corresponding depth image and that the file exists
-        assert (
+        mismatch = (
             samples.loc[samples.label_index == LabelName.ABNORMAL]
             .apply(lambda x: Path(x.image_path).stem in Path(x.depth_path).stem, axis=1)
             .all()
-        ), "Mismatch between anomalous images and depth images. Make sure the mask files in 'xyz' \
-            folder follow the same naming convention as the anomalous images in the dataset \
-            (e.g. image: '000.png', depth: '000.tiff')."
+        )
+        if not mismatch:
+            msg = """Mismatch between anomalous images and depth images. Make sure the mask
+              files in 'xyz' folder follow the same naming convention as the anomalous
+              images in the dataset (e.g. image: '000.png', depth: '000.tiff')."""
```

Comment:
You will need to split this line as it is longer than 120 chars, which will fail the CI. Something like

```python
msg = (
    "Mismatch between anomalous images and depth images. Make sure the mask"
    "files in 'xyz' folder follow the same naming convention as the anomalous"
    "images in the dataset (e.g. image: '000.png', depth: '000.tiff')."
)
```

Reply:
oh I realized that you used """. It might work in this case.